src/LimitCalib.cpp

```diff
@@ -22,7 +22,7 @@ void callback(motorid_t motor, const types::DataPoint<LimitSwitchData>& data) {
 };
 
 void cleanup(int signum) {
-	log(LOG_ERROR, "Interrupted!\n");
+	LOG_F(ERROR, "Interrupted!\n");
```

Comment:
I'll just put the note here instead of everywhere, but I believe loguru automatically appends a newline, so we can remove the newline at the end of all of our logs.